PcdcAnalysisTools/blueprint/routes/views/survival/__init__.py

```diff
@@ -61,21 +70,24 @@ def get_result():
 OVERALL_TIME_VAR = "survival_characteristics.age_at_lkss"
 
 
-def fetch_data(filters, efs_flag):
+def fetch_data(config, filters, efs_flag):
     status_str, status_var, time_var = (
         (EVENT_FREE_STATUS_STR, EVENT_FREE_STATUS_VAR, EVENT_FREE_TIME_VAR)
         if efs_flag
         else (OVERALL_STATUS_STR, OVERALL_STATUS_VAR, OVERALL_TIME_VAR)
     )
 
-    filters.setdefault("AND", [])
-
     guppy_data = utils.guppy.downloadDataFromGuppy(
         path=capp.config['GUPPY_API'] + "/download",
         type="subject",
         totalCount=100000,
         fields=[status_var, time_var],
-        filters=filters,
+        filters={
+            "AND": [
+                {"IN": {"consortium": config.get('consortium')}},
+                filters
+            ]
+        },
```

Comment:
Actually just noticing this. it should be conditional, not applied at priory. If the config["consortia"] is empty this should not be applied.